h/services/oauth.py

```diff
@@ -148,7 +149,9 @@ def create_token(self, user, authclient):
         :rtype: h.models.Token
         """
         token = models.Token(userid=user.userid,
+                             value=security.token_urlsafe(),
```

Comment:
I think it would be a good idea to have a prefix on this token type too so we can distinguish one random string of characters from another when debugging. Perhaps also on the refresh token.